kong/plugins/oauth2/migrations/cassandra.lua

```diff
@@ -103,5 +103,15 @@ return {
         end
       end
     end
+  },
+  {
+    name = "2016-07-15-oauth2_code_credential_id",
+    up = [[
+      TRUNCATE oauth2_authorization_codes;
```

Comment:
Authorization codes are different than access tokens. They are supposed to be consumed immediately and they expire after 5 minutes. Not a big deal.